tsconfig.docker.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Why do we need this file since we already have it ?

Reply:
we need this for apollo-server which has node environment and need another config.